Week1/homework/exercise3.js

```diff
@@ -0,0 +1,8 @@
+let numberx = []
+console.log ("x is undefined because I didn't atribute any value to it");
+console.log (numberx);
+var numberx= 3;
+console.log ("x is equal to 3");
+console.log (numberx);
```

Comment:
Ah. There seems to be a mistake.
- You declared the variable `numberx` and initialised it with an empty array(`[]`). Thus, it is not **undefined** anymore but has a value which is an empty array.
- Once you have declared a variable, you need not re-declare it as you do in line 4. Also, try to avoid using `var` keywords while declaring variables.